iam-login-service/src/main/webapp/WEB-INF/views/iam/login.jsp

```diff
@@ -68,9 +72,18 @@
 
       <div>
         <input type="hidden" name="${_csrf.parameterName}" value="${_csrf.token}"> 
-        <input type="submit" class="btn btn-primary btn-block" value="Login" name="submit" class="form-control">
+        <input type="submit" class="btn btn-primary btn-block" value="${loginPageConfiguration.loginButtonText}" 
+          name="submit" class="form-control">
       </div>
```

Comment:
robot-testsuite uses `xpath=//input[@value='Login']` to click the button and login.
so it could be better to use an id and update the testsuite